devices/generic/items/state_power_item.json

```diff
@@ -15,7 +15,7 @@
     "at": "0x050b",
     "cl": "0x0b04",
     "ep": 0,
-    "eval": "if (Attr.val != -32768 && Attr.val != 32768) { Item.val = Attr.val; }"
+    "eval": "if (Attr.val != -32768 && Attr.val != 32768) { Item.val = Attr.val * R.item('state/power_multiplier').val / R.item('state/power_divisor').val; }"
```

Comment:
- Might want to add `Math.round()` to make sure the reported value is integer.
- Signed `s16` supports values from -32768 to 32767, so `Attr.val != 32768` is superfluous.  Typically the lowest value, -32768 is used to indicate an invalid value.  
Unsigned `u16` supports values from 0 to 65535, and uses for 65535 as invalid value.

Reply:
> Signed s16 supports values from -32768 to 32767, so Attr.val != 32768 is superfluous.

Actually I didn't change that. [This code is already in master](https://github.com/search?q=repo%3Adresden-elektronik%2Fdeconz-rest-plugin%20%22Attr.val%20!%3D%2032768%22&type=code). Do we need to clean it up?

Reply:
Yes, I saw that.  Better cleanup while you’re touching it anyways.  Sorry to put this on you.

Reply:
> Might want to add Math.round() to make sure the reported value is integer.

The reported value is not required to be Integer - am I wrong? JavaScript Number(s) are always floating point. Normally it is up to the UI to take care about display and formatting. \
While I was testing with the plug's, I already had problems reporting the standby demand (it was always `0W` in Phoscon, while the RAW value was `4` in deCONZ). I can't tell if that's related to this specific code, but for sure if we reduce resolution here, we throw away information and my end up not being able to show fractions.

Reply:
> Yes, I saw that. Better cleanup while you’re touching it anyways. Sorry to put this on you.

Because the issue is not only related to the files I am touching here, I would like to issue another PR for the cleanup only, to not mix up topics. (It should not matter if that one then gets merged before or after this one, since we can rebase)

Reply:
Since the cleanup was moved to a dedicated PR and doesn't block us here, is there anything which prevents us continuing? (I don't want to leave this stalled, since it already took multiple days of work)

If the calculation is fine, let's resolve this conversation.

Reply:
@ebaauw bump